packages/admin-ui/src/components/Modal/components/ModalFooter.tsx

```diff
@@ -17,51 +17,62 @@ import { useModalContext } from './ModalContext'
  * </Modal>
  * ```
  */
-export const ModalFooter = jsx('footer')(
-  {
-    borderTopWidth: '1px',
-    display: 'flex',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    borderTop: '$neutral',
-    position: 'sticky',
-    bottom: 0,
-    left: 0,
-    right: 0,
-    bg: '$primary',
-    padding: 6,
-    flexDirection: ['column-reverse', 'column-reverse', 'row'],
-    '>button': {
-      width: ['full', 'full', 'inherit'],
-    },
-    '* + button': {
-      marginLeft: [0, 0, 4],
-      marginBottom: [4, 4, 0],
-    },
-  },
-  {
-    useOptions(_, props) {
-      const { csx, ...footerProps } = props
-      const { size } = useModalContext()
+export const ModalFooter = createComponent<'footer'>((props) => {
+  const { size } = useModalContext()
+
+  return useElement('footer', {
+    ...props,
+    baseStyle: {
+      borderTopWidth: '1px',
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      borderTop: '$neutral',
+      position: 'sticky',
+      bottom: 0,
+      left: 0,
+      right: 0,
+      bg: '$primary',
+      padding: 6,
+      flexDirection: ['column-reverse', 'column-reverse', 'row'],
```

Comment:
is this for the "responsiveness array"?

Reply:
I guess you should change it for the responsive aliases

Reply:
And we have a bug about it: https://vtex.slack.com/archives/C01DVTFA4VA/p1658162740473519.

Reply:
yes! I've probably missed it :squirrel: 
I'll change, good catch!